setup.py

```diff
@@ -0,0 +1,26 @@
+from setuptools import setup, find_packages
+
+with open('requirements.txt') as f:
+    required = f.read().splitlines()
+
+setup(
+    name='insight_docclean',
+    version='0.1',
+    packages=find_packages(),
+    install_requires=required,
+    tests_require=['pytest'],
+    zip_safe=False,
+    url='https://github.com/devanshkv/insight_docclean/',
+    license='GNU Lesser General Public License v3.0',
+    author='Devansh Agarwal',
+    author_email='devansh.kv@gmail.com',
+    description='Clean your document images'
+classifiers = [
```

Comment:
Nitpick linting issue: should be `classifier=[`. Also missing indent

Reply:
Fixed with commit 4d6d6c3.